test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1717,7 +1719,7 @@ public void testPertableClasspath() throws Exception {
     fooFilterJar.deleteOnExit();
 
     File fooConstraintJar = File.createTempFile("FooConstraint", ".jar", new File(rootPath));
-    FileUtils.copyInputStreamToFile(this.getClass().getResourceAsStream("/FooConstraint.jar"),
+    FileUtils.copyInputStreamToFile(this.getClass().getResourceAsStream("/FooConstraint_2_1.jar"),
```

Comment:
I assume this new jar has constraints using the new public API package? I'm not sure it matters for this test. We should support older constraints as well, so this test should pass whether or not the jar contains classes that implement the new or old constraint interface.

Reply:
Yes - this uses the new interface.  When I tested with something built with the older interface it failed.  I think one way to handle this is to add a tests that tests a jar built with the each version - step one was just to see if this would get the test to pass (which it does now) in case that would help point a way forward - this was the motivation to be able to build jars for multiple versions.